### PR TITLE
refactor(hosting.general.informations): use oui component & class

### PR DESCRIPTION
**Pull request by TedKac**
## refactor(hosting.general.informations): use oui-header & oui-tile__term

### CODE REVIEW

File: client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -15,12 +15,12 @@
             <div class="row d-md-flex" data-ng-if="!hosting.isExpired">
                 <div class="col-sm-6">
                     <div class="oui-tile mb-5 h-100">
-                        <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_general_informations"></h3>
+                        <h4 class="oui-header_4 oui-tile__title" data-i18n-static="hosting_dashboard_general_informations"></h4>
```
Comment (JeremyDec) [inline client/app/hosting/general-informations/GENERAL_INFORMATIONS.html]: Are you sure you have a parent `h3` tag ?

File: client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 2]
```diff
@@ -143,14 +143,14 @@ <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_general_informat
                 </div>
                 <div class="col-sm-6">
                     <div class="oui-tile mb-5 h-100">
-                        <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_configuration"></h3>
+                        <h4 class="oui-header_4 oui-tile__title" data-i18n-static="hosting_dashboard_configuration"></h4>
```
Comment (JeremyDec) [inline client/app/hosting/general-informations/GENERAL_INFORMATIONS.html]: Same question

File: client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 3]
```diff
@@ -429,12 +429,12 @@ <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_configuration"><
 
                 <div class="oui-tile mb-5"
                      data-ng-if="!hosting.isExpired">
-                    <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_access"></h3>
+                    <h4 class="oui-header_4 oui-tile__title" data-i18n-static="hosting_dashboard_access"></h4>
```
Comment (JeremyDec) [inline client/app/hosting/general-informations/GENERAL_INFORMATIONS.html]: Same question

File: client/app/hosting/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (TedKac) [reply client/app/hosting/general-informations/GENERAL_INFORMATIONS.html]: Fixed by using h3 and class oui-header_4 😄

**Pull request merged**